SeleniumDemoQA/SeleniumDemoQA/Pages/FormPage.cs

```diff
@@ -0,0 +1,124 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeleniumDemoQA.Pages
+{
+    internal class FormPage
+    {
+        public IWebDriver _driver;
+        public IJavaScriptExecutor _js;
+        By confirmationModalElement = By.Id("example-modal-sizes-title-lg");
+
+        public FormPage(IWebDriver driver)
+        {
+            _driver = driver;
+            _js = (IJavaScriptExecutor)_driver;
+        }
+
+        public void ScrollTo(IWebElement element)
+        {
+            _js.ExecuteScript("arguments[0].scrollIntoView(true);", element);
+        }
+
+        public void FillInput(By selector, string value)
+        {
+            var webElement = _driver.FindElement(selector);
+            ScrollTo(webElement);
+            webElement.SendKeys(value);
+        }
+        public void ClickElement(By selector)
+        {
+            var webElement = _driver.FindElement(selector);
+            ScrollTo(webElement);
+            webElement.Click();
+        }
+        public void FillInputAndEnter(By selector, string value)
+        {
+            var webElement = _driver.FindElement(selector);
+            ScrollTo(webElement);
+            webElement.SendKeys(value);
+            webElement.SendKeys(Keys.Enter);
+        }
+        public void ClickWithoutScroll(By selector)
+        {
+            var webElement = _driver.FindElement(selector);
+            webElement.Click();
+        }
+        public void SelectText(By selector, string text)
+        {
+            var webElement = new SelectElement(_driver.FindElement(selector));
+            webElement.SelectByText(text);
+        }
+
+        public string GetGss(string id)
```

Comment:
better to name GetBorderColor because it returns only one particular Css value but not any valie

Reply:
In order to avoid multiple methods like GetBorderColorById(string id), GetBorderColorByCss(string css), GetBorderColorByXpath(string xPath), GetBorderColorByClassName(string className), GetBorderColorByText(string text), it is better to have only one method
GetBorderColor(By selector)
This is valid for methods below as well. Instead of calling FillInputById you can directly call FillInput. In any case you have to specify element Id, which id Page detail, so on this stage we can use calls like formPage.FillInput(By.Id("firstName"), etc.